index.bs

```diff
@@ -676,6 +680,29 @@ Issue: Parse |value|["`filter_data`"].
 
 Issue: Parse |value|["`aggregation_keys`"].
 
+<h3 id="check-pending-destination-limit">Checking pending destination limit</h3>
+
+To <dfn>check if an [=attribution source=] meets the pending destination limit</dfn> given an
+[=attribution source=] |source|, run the following steps:
+
+1. Let |pendingSources| be all entries in the [=attribution source cache=] where all of the following are true:
+     * entry's [=attribution source/source site=] and |source|'s [=attribution source/source site=]
+        are equal.
+     * entry's [=attribution source/reporting endpoint=] and |source|'s
+        [=attribution source/reporting endpoint=] are [=same origin=].
+     * entry's [=attribution source/number of event-level reports=] is equal to 0.
+1. Let |distinctDestinations| be a new [=set/is empty|empty=] [=set=].
+1. For each [=attribution source=] |pendingSource| of |pendingSources|:
+    1. If |source|'s [=attribution source/attribution destination=] and |pendingSource|'s
+        [=attribution source/attribution destination=] are equal, then return true.
+    1. If |distinctDestinations| [=set/contains=] |pendingSource|'s
```

Comment:
I think this adds unnecessary work, set/append should implicitly avoid the double counting.